### PR TITLE
Cargo: misc package upgrades

### PR DESCRIPTION
**Pull request by LyleScott**
These are misc package upgrades I've made in the scope of a feature branch of mine. I've rolled with these changes for more of the dev cycle, so they seem innocent enough.

> #46  #37  are parent branches of this branch. This was to avoid code slosh from other PRs that this PR depends on (ie, upgrading k8s, gstreamer, tokio, which are all already done in other PRs).

### CODE REVIEW

**Review by LourdesMoran — APPROVED** [review]
lgtm

**Comment by LyleScott** [comment]
funfact: don't upgrade DSSIM library. i tried this and it sent me for a loop for a while! There's been a major version bump and the algorithm is supposed to be better, but it changes the scoring which throws off the thresholds me set for finding a match. To upgrade, we'll need to test thoroughly and rescore/find the sweetspot threshold.

File: hawkeye-core/Cargo.toml [hunk 1]
```diff
@@ -7,10 +7,9 @@ edition = "2018"
 
 [dependencies]
 log = "0.4"
-ureq = "1.4"
 color-eyre = "0.5"
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
+serde_json = "1.0.73"
```
Comment (LyleScott) [inline hawkeye-core/Cargo.toml:12]:
```suggestion
serde_json = "1.0"
```

File: hawkeye-worker/Cargo.toml [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-
 [dependencies]
 hawkeye-core = { path = "../hawkeye-core" }
 image = "0.23"
 glib = "0.14.8"
 gstreamer = "0.17.4"
 gstreamer-app = "0.17.2"
 derive_more = "0.99.17"
-dssim = "2.11"
+dssim = "3.1.0"
```
Comment (LyleScott) [inline hawkeye-worker/Cargo.toml:16]:
```suggestion
dssim = "2.11"
```

See comment on main PR about this change... and why it needed to be the old version.

**Review by rafaelcaricio — APPROVED** [review]

**Comment by rafaelcaricio** [comment]
@LyleScott Maybe we could also upgrade the Rust version we are using https://github.com/cbsinteractive/hawkeye/blob/fcc963e860485d9d6ae3932767a72c3e1eaffe44/worker.Dockerfile#L1 ?

**Comment by LyleScott** [comment]
> @LyleScott Maybe we could also upgrade the Rust version we are using
> 
> https://github.com/cbsinteractive/hawkeye/blob/fcc963e860485d9d6ae3932767a72c3e1eaffe44/worker.Dockerfile#L1

Absolutely :) By this time, you've seen https://github.com/cbsinteractive/hawkeye/pull/48 so I assume we're all good here.

I would, though, like to explore even using 2021 instead of 2018 at some point. I'll give that a go when i'm waiting for builds sometime.

**Pull request merged**